include/libserver/Util.hpp

```diff
@@ -71,6 +72,10 @@ struct WinFileTime {
 WinFileTime UnixTimeToFileTime(
   const std::chrono::system_clock::time_point& timePoint);
 
+//! Resolve hostname to address
+namespace asio = boost::asio;
```

Comment:
Move the alias above the `UnixTimeToFileTime` function.

Reply:
Also remove the comment, it doesnt make sense.

Reply:
done